doc/files.md

```diff
@@ -26,40 +26,42 @@ Linux    | `$HOME/.bitcoin/`
 macOS    | `$HOME/Library/Application Support/Bitcoin/`
 Windows  | `%APPDATA%\Bitcoin\` <sup>[\[1\]](#note1)</sup>
 
-2. The non-default data directory path can be specified by `-datadir` option.
+2. The data directory path can be specified with `-datadir` option.
 
 3. All content of the data directory, except for `bitcoin.conf` file, is chain-specific. This means the actual data directory paths for non-mainnet cases differ:
 
-Chain option        | Data directory path
---------------------|--------------------
-no option (mainnet) | *path_to_datadir*`/`
-`-testnet`          | *path_to_datadir*`/testnet3/`
-`-regtest`          | *path_to_datadir*`/regtest/`
+Chain option                   | Data directory path
+-------------------------------|------------------------------
+`-chain=main` (by default)     | *path_to_datadir*`/`
+`-chain=signet` or `-signet`   | *path_to_datadir*`/signet/`
+`-chain=test` or `-testnet`    | *path_to_datadir*`/testnet3/`
+`-chain=regtest` or `-regtest` | *path_to_datadir*`/regtest/`
 
 ## Data directory layout
 
 Subdirectory       | File(s)               | Description
 -------------------|-----------------------|------------
-`blocks/`          |                       | Blocks directory; can be specified by `-blocksdir` option (except for `blocks/index/`)
+`blocks/`          |                       | Blocks directory. Path can be specified with `-blocksdir` option (except for `blocks/index/`)
 `blocks/index/`    | LevelDB database      | Block index; `-blocksdir` option does not affect this path
 `blocks/`          | `blkNNNNN.dat`<sup>[\[2\]](#note2)</sup> | Actual Bitcoin blocks (in network format, dumped in raw on disk, 128 MiB per file)
 `blocks/`          | `revNNNNN.dat`<sup>[\[2\]](#note2)</sup> | Block undo data (custom format)
-`chainstate/`      | LevelDB database      | Blockchain state (a compact representation of all currently unspent transaction outputs and some metadata about the transactions they are from)
+`chainstate/`      | LevelDB database      | Blockchain state (a compact representation of all currently unspent transaction outputs (UTXO) and some metadata about the transactions they are from)
```

Comment:
s/UTXO/UTXOs/ and omit parasitic "some"

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/20076#issuecomment-703294876).